src/zst_expr.rs

```diff
@@ -0,0 +1,141 @@
+/*!
+ * Behold, the smoking remains of some ravaged Rust code, the result of an
+ * invocation gone awry, for Yandros had been awakened from their slumber.
+ *
+ * The Intelligence Dampening Sphere, thinking it could overtake Rust and its
+ * constraints, had engaged in a devastating battle.
+ *
+ * But the Crab was no easy creature to subdue, and no matter how many
+ * cursed poorman specialization stabs were taken against it, the Crab remained,
+ * unscathed.
+ *
+ * Unwilling to admit defeat, the challenger, desperate, started chanting an
+ * ancient incantation from the forbidden times, unwittingly dooming us all:
+ *
+ * ```rs
+ * #[allow(unstable_features)]
+ * #[feature(specialization)]
+ * ```
+ *
+ * The Crab immediately unleashed an ear-piercing shriek of pain, after which
+ * there was no sound: complete and utter silence.
+ *
+ * ---
+ *
+ * Tread carefully among these paths, adventurer, and forgo any hope for sane
+ * Rust code.
```

Comment:
May I submit this to [TWiR Quote of the Week](https://users.rust-lang.org/t/twir-quote-of-the-week/328/1486)? :smile:

Reply:
If you want 😄 (it may be a bit long though 😅)